pkg/virt-launcher/virtwrap/live-migration-target_test.go

```diff
@@ -53,8 +54,10 @@ var _ = Describe("client", func() {
 		os.RemoveAll(shareDir)
 	})
 
-	It("Should monitor the end of a migration", func() {
-		var domainJobType = libvirt.DOMAIN_JOB_BOUNDED
+	DescribeTable("Should monitor the end of a migration", func(type1, type2 libvirt.DomainJobType, op1, op2 libvirt.DomainJobOperationType, err1, err2 error) {
```

Comment:
err2 is always nil :) Any missing entries?

Reply:
That's correct, no test sets the second error to anything, I left it for completeness / future use. Would you prefer I remove it?

Reply:
Naa, maybe we can add a todo

Reply:
Gone, no need to ever test with a second error actually, it'd just delay the event.